internal/rbd/globals.go

```diff
@@ -89,6 +93,21 @@ func SetGlobalBool(name string, value bool) {
 	}
 }
 
+// SetGlobalString provides a way for the rbd-driver to configure global variables
+// in the rbd package.
+//
+// TODO: these global variables should be set in the ControllerService and
+// NodeService where appropriate. Using global variables limits the ability to
+// configure these options based on the Ceph cluster or StorageClass.
+func SetGlobalString(name, value string) {
+	switch name {
+	case k8s.GetCOKeyName():
```

Comment:
why is the cluster name specific to Kubernetes? Should this not be in a generic `util` function?

Reply:
Currently we are using k8s specific naming for all the rest 6 keys
```
csiParameterPrefix = "csi.storage.k8s.io/"                               
                                                                         
// PV and PVC metadata keys used by external provisioner as part of      
// create requests as parameters, when `extra-create-metadata` is true.  
pvcNameKey      = csiParameterPrefix + "pvc/name"                        
pvcNamespaceKey = csiParameterPrefix + "pvc/namespace"                   
pvNameKey       = csiParameterPrefix + "pv/name"                         
                                                                         
// snapshot metadata keys.                                               
volSnapNameKey        = csiParameterPrefix + "volumesnapshot/name"       
volSnapNamespaceKey   = csiParameterPrefix + "volumesnapshot/namespace"  
volSnapContentNameKey = csiParameterPrefix + "volumesnapshotcontent/name"
```

just keeping the naming aligned with the rest, hence this was kept in k8s package.

Reply:
If you insist, I can move it to utils, but I don't know what prefix should we choose?